### ISSUE TITLE
Ortssuche 

### ISSUE THREAD

**Issue by OpenGreenStreet** [commons-app/apps-android-commons#5719]
### What is the user problem or growth opportunity you want to see solved?

Eine Ortssuche über den Namen und auch über die GPS-Koordinaten (Thale 51.750000,11.044100) wäre super 

### How do you know that this problem exists today? Why is this important?

So kann ich Bilder nach pflegen, die zuvor an Orten "in der Nähe" aufgenommen habe.

### Who will benefit from it?

All users

**Comment by nicolas-raoul** [comment]
Not sure I understand.

Would this be a new feature in the Nearby UI, or in the Upload Wizard's depiction search UI, or somewhere else?

Please list steps of how users would use this feature. Feel free to post screenshots with drawings added, if that can help.

Thanks! :-)

**Comment by OpenGreenStreet** [comment]
![Screenshot_20240512-071111](https://github.com/commons-app/apps-android-commons/assets/76477269/250ff280-f7b5-4c94-80df-e20a601f3081)
![Screenshot_20240512-071208](https://github.com/commons-app/apps-android-commons/assets/76477269/24f438ab-ea27-45c3-b575-856aa3465450)
![Screenshot_20240512-071513](https://github.com/commons-app/apps-android-commons/assets/76477269/e2f8b56a-4879-4bec-8d55-2686ba58d8cb)
![Screenshot_20240512-071548](https://github.com/commons-app/apps-android-commons/assets/76477269/84135693-bde2-462c-a78b-1a52877dd71e)
![Screenshot_20240512-071629](https://github.com/commons-app/apps-android-commons/assets/76477269/9dcf1e0e-a1c5-46e5-b2ea-e865deb0a599)
![Screenshot_20240512-071732](https://github.com/commons-app/apps-android-commons/assets/76477269/f04fd477-f322-4b1f-9345-a85ec68e93bd)

**Comment by OpenGreenStreet** [comment]
Do these screenshots help for a better understanding?

**Comment by OpenGreenStreet** [comment]
![Screenshot_20240512-071111](https://github.com/commons-app/apps-android-commons/assets/76477269/9d37292c-d067-4ee1-952e-3bd0006e7d4a)
![Screenshot_20240512-071208](https://github.com/commons-app/apps-android-commons/assets/76477269/68d4e923-f2ac-461e-b36c-b2c0a5e81959)
![Screenshot_20240512-071513](https://github.com/commons-app/apps-android-commons/assets/76477269/0d4d1c1a-6a83-40e9-b6fa-e1225ab6ad6d)
![Screenshot_20240512-071629](https://github.com/commons-app/apps-android-commons/assets/76477269/11ee1a7a-7d4f-48c4-86ee-2a173ef0d2f5)
![Screenshot_20240512-071732](https://github.com/commons-app/apps-android-commons/assets/76477269/f9549918-780d-4831-ae65-096931f1ce64)
![Screenshot_20240512-071548~2](https://github.com/commons-app/apps-android-commons/assets/76477269/f4946b99-fc75-4ce7-8e2a-c7f481ebcdd9)

**Comment by nicolas-raoul** [comment]
Let me summarize:
- You have a picture
- You want to upload it to Commons
- You do not know whether that picture depicts a particular Wikidata item

How about adding a `Select on map` button on the Upload Wizard's depiction selection activity? It would show a map similar to the Nearby map (but with "Needs photo" disabled by default), and let the user pick depiction(s).

**Comment by OpenGreenStreet** [comment]
- I know that the picture is being searched for 
- I have taken the picture you are looking for 
- but I am now many kilometers away and have difficulty finding the location on the map (but I know the name of the place and/or its coordinates)

**Comment by nicolas-raoul** [comment]
I understand better now, thanks!

In such a situation, the solution I suggested in my previous comment would work well if the picture has EXIF location, as it would center the map on the picture's location.

**Comment by OpenGreenStreet** [comment]
Yes, that could be a good idea. Another might be the following:
From my point of view, only one suggestion is currently made as to which Wikidata object might fit if EXIF information is available.
If a list were offered here within a limited radius, the appropriate one could be selected. (In Stolberg (Harz), for example, about 10 houses located directly next to each other are being searched for. It is probably more of a coincidence if the right one is offered).

**Comment by nicolas-raoul** [comment]
Yes, the `Is this a picture of X` popup could have a button "See more nearby places" that would open the same map as the `Select on map` button.

**Comment by OpenGreenStreet** [comment]
Yes, that sounds very good.
Irrespective of this, I would like to see an (additional) "go to" function on the map
